Snakefile

```diff
@@ -1,18 +1,25 @@
 from benchutils import metrics, plotting, transformers
 from itertools import product
+import os
+import shutil
 
 # TODO move to config
 filename = "anonymous_reads"
 (SIM, DT, NUM, EXT) = glob_wildcards("data/simulations/{simname}/{datetime}_sample_{sample_num}/reads/" + filename + ".{extension}")
 
-# this sorting does not account for different DT's and sample_nums for different simulations...
+# parse methods for settings
+METHODS = []
+for method in config["params"]:
+    if isinstance(config["params"][method], list):
+        settings = config["params"][method]
+        METHODS.extend(method + '_' + str(i) for i in list(range(len(settings))))
```

Comment:
What is this `list` for?

Reply:
Good catch, I think its a holdover from when I was editing this list comprehension. Will remove. Thanks!